lib/tasks/screenshots.rake

```diff
@@ -41,7 +41,7 @@ if Rails.env.development?
     screenshot_file = 'about_us.png'
     page.save_screenshot(screenshot_file)
     puts "Saved screenshot #{screenshot_file}"
-    spectre.submit_test('About Us', 'Phantom', 'OSX', 1024, File.new(screenshot_file, 'rb'))
+    spectre.submit_test('About Us', 'Phantom', 'OSX', 1024, File.new(screenshot_file, 'rb'), "", "90%")
```

Comment:
yeah, this is pretty ugly. I was considering replacing all the args with an options object but worried we would need to fix our existing clients.